packages/file_selector/file_selector_platform_interface/LICENSE

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
```

Comment:
Could you pull a new copy of this license from one of the other license files in the repo? I recently standardized the formatting in all of them (but haven't yet written an automated check for PRs adding new ones)

Reply:
Sounds good